core/templates/dev/head/pages/admin-page/misc-tab/admin-misc-tab.directive.ts

```diff
@@ -142,6 +144,17 @@ angular.module('oppia').directive('adminMiscTab', [
           ctrl.dataExtractionQueryStatusMessage = message;
         };
 
+
+        ctrl.sendDummyMailToAdmin = function() {
+          $http.post(SEND_DUMMY_MAIL_HANDLER_URL, {
+            action: 'send_mail_to_admin'
```

Comment:
I don't understand. Why do you need this action variable in the first place? What happens if you omit it?

Reply:
When I try to omit it the program is throwing a error asking for another parameter to be sent.

Reply:
What is the error? If you see an error, in general, please dig further into it to understand what is going on. In general POST commands do not need an action parameter.

Reply:
The post request in the python test file requested for a payload parameter and so I created a dummy parameter to send and to make it consistent I also changed it in the ts file. Now I have made changes to send a empty object.